codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -243,10 +245,10 @@ sector = Sector callStack
 
 -- | A piece of text
 text :: HasCallStack => Text -> Picture
-text = Text callStack Plain Serif
+text = Text callStack
 
 styledText :: HasCallStack => TextStyle -> Font -> Text -> Picture
-styledText = Text callStack
+styledText = StyledText callStack Plain Serif
```

Comment:
I don't think you want Plain and Serif here.

Admittedly, this is a bit confusing.  There's some eta expansion needed to see all the parameters.  That comes from the fact that these lines of code:

    f = g
    f x = g x

mean the same thing.  Technically, the first line says that f and g are the same, while the second says that they have the same result for any input.  But since a function is *completely* determined by its result for each input, that's enough to conclude that they are the same anyway!  This fact is sometimes called "eta equivalence", and converting an equation `f = g` into the equation `f x = g x` is called "eta expansion".

The same thing was happening here.  The original code was `styledText = Text callStack`.  But after an eta expansion, that's the same thing as `styledText style = Text callStack style`, and two MORE eta expansions gives you `styledText style font txt = Text callStack style font txt`.  Now you can see all four arguments to the constructor.

The original style (`f = g` or `styledText = Text callStack`) is nice because it lets you talk about entire functions, instead of their specific values at specific parameters.  But I suppose it's not so nice if you aren't familiar with it.

Reply:
Thank you so much for this!